src/lib-python/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ list( APPEND libs
   collect_sources.py
   meteo_utils.py
   orddicts.py
+  pytspack.py 
+  tspack.cpython-37m-x86_64-linux-gnu.so
```

Comment:
This is bad, no? this means it will ONLY work for python 3.7 on X86 64bit linux

Reply:
I agree. It will causes incompatibility issue on different machine while running this python code. So I removed codes for AOD interpolations. Thanks.